README.md

```diff
@@ -288,6 +288,24 @@ dartdoc is currently documenting.  This can lead to inconsistent behavior betwee
 packages, especially if different command lines are used for dartdoc.  It is recommended to use collision-resistant
 naming for any macros by including the package name and/or library it is defined in within the name.
 
+### Documentation imports (`/// @docImport`)
+
+Libraries that are only referenced in documentation comments (`[Future]`) can be imported with a
+`/// @docImport '<uri>'` comment on the `library` element, like:
+
+```dart
+/// @docImport 'dart:async';
+/// @docImport 'package:flutter/element.dart' show Element;
```

Comment:
'show' is not supported. (yet?)

Reply:
Exactly. Noting the umbrella issue for it does mention this was a goal https://github.com/dart-lang/sdk/issues/56186.